pkg/kubelet/runtime.go

```diff
@@ -34,6 +35,7 @@ type runtimeState struct {
 	storageError             error
 	cidr                     string
 	healthChecks             []*healthCheck
+	rtHandlers               map[string]kubecontainer.RuntimeHandler
```

Comment:
nit:  `runtimeHandlers`

Reply:
I've named it `rtHandlers` to not clash with the getter `runtimeHandlers()`.  Should I use `getRuntimeHandlers()`?